packages/next/src/shared/lib/modern-browserslist-target.d.ts

```diff
@@ -0,0 +1,10 @@
+// Keep in sync with the `.js` file.
+declare const MODERN_BROWSERSLIST_TARGET: [
+  'chrome 64',
+  'edge 79',
+  'firefox 67',
+  'opera 51',
+  'safari 12'
+]
```

Comment:
```suggestion
declare const MODERN_BROWSERSLIST_TARGET: string[]
```

Could we change this to `string[]` so we don't need to keep it in sync?

Reply:
I like the explicit reference more as it allows you to peek the values:
<img width="796" alt="CleanShot 2023-04-24 at 09 22 47@2x" src="https://user-images.githubusercontent.com/6324199/233926766-70ff4fc6-b544-485d-a3dc-5eb8576b1c3c.png">